lib/wallet/src/Cardano/Wallet/DB.hs

```diff
@@ -28,34 +27,38 @@ module Cardano.Wallet.DB
     , DBWalletMeta (..)
     , DBDelegation (..)
     , DBTxHistory (..)
-    , DBPendingTxs (..)
     , DBPrivateKey (..)
     , mkDBLayerFromParts
 
-      -- * Errors
-    , ErrBadFormat(..)
-    , ErrWalletAlreadyExists(..)
-    , ErrNoSuchTransaction (..)
-    , ErrRemoveTx (..)
-    , ErrPutLocalTxSubmission (..)
-    , getInSubmissionTransaction_
     , hoistDBLayer
+    , updateSubmissions
```

Comment:
```suggestion
```

This helper function is very effective in the present module, but I would prefer not to export it — the goal of `DBVar` is to separate IO from non-IO functions, so helpers like `adjustNoSuchWallet` are good exports, but functions like `updateSubmission` that mix the `walletsDB` with the pure functions again would be somewhat contrary to the goal.